packages/beacon-node/src/network/processor/index.ts

```diff
@@ -48,6 +51,31 @@ const executeGossipWorkOrder = Object.keys(executeGossipWorkOrderObj) as (keyof
 // TODO: Arbitrary constant, check metrics
 const MAX_JOBS_SUBMITTED_PER_TICK = 128;
 
+// How many attestations (aggregate + unaggregate) we keep before new ones get dropped.
+const MAX_QUEUED_UNKNOWN_BLOCK_GOSSIP_OBJECTS = 16_384;
+
+// We don't want to process too many attestations in a single tick
+// As seen on mainnet, utilization rate for attestation topic is 80_000% (800x) - 90_000% (900x)
```

Comment:
We should stop looking at the utilization rate as it does not really make sense and track the concurrency metric

https://github.com/ChainSafe/lodestar/blob/887064d7f497adf123a3839ec4c71c323c24bbdd/packages/beacon-node/src/metrics/metrics/lodestar.ts#L269-L270

Reply:
<img width="1291" alt="Screenshot 2023-04-05 at 08 44 16" src="https://user-images.githubusercontent.com/10568965/229959718-d6b5901f-4db6-41e5-9e57-13df1466c823.png">

The concurrency metric ranges from 1000 to 2000 so I'd make it a little bit conservative, 1024, too. Will change the comment to base on this metric.